src/Model/MethodGo.cs

```diff
@@ -592,7 +592,7 @@ public bool IsPageable
                 // if the nextLinkName field in the swagger has a null value ("nextLinkName": null)
                 // then don't treat this operation as pageable.
                 var pageableExtension = Extensions[AzureExtensions.PageableExtension] as JContainer;

```

Comment:
can this be null ?

Reply:
No, if it was null the previous if clause would cause early return.